test/unit/specs/store/getters.spec.js

```diff
@@ -63,7 +63,7 @@ describe('store', () => {
           covid19: {filters: ['covid19']},
           showCountryFacet: true
         }
-        expect(getters.biobankRsql(state)).to.equal('country=in=(AT,BE);covid19biobank=in=(covid19)')
+        expect(getters.biobankRsql(state)).to.equal('country=in=(AT,BE);(covid19biobank==covid19)')
```

Comment:
for posterity, and future changes, update the test to one with two selected criteria so that the AND condition on the selections comes out more clearly.